### ISSUE TITLE
Cheat sheet for operators

### ISSUE THREAD

**Issue by marionschleifer** [hasura/graphql-engine#2988]
Create cheat sheet (could be a guide in the docs https://docs.hasura.io/1.0/graphql/manual/guides/index.html) for operators (for example: `_or`, `_eq`).

We should have a list of operators and corresponding examples for each.

**Comment by cobwebsonsale** [comment]
@marionschleifer I think the doc on Filter (https://docs.hasura.io/1.0/graphql/manual/queries/query-filters.html) suffices as a cheat sheet for these operators. It has ample examples for an operator too.
The only ones I find missing from the those in this issue are `_ceq` and `_cne`. Such column comparators may be added as a new row on the same page. What do you think?

**Comment by greenorb007** [comment]
Please assign this to me i am really a newbie and really want to work on it, and please guide me little on this.
Thank you

**Comment by parikshitgupta1** [comment]
Assign me on this aswell.

**Comment by marionschleifer** [comment]
@cobwebsonsale I agree. Let's add the missing operators for now. I adapted the description.

If we find a good place for a cheat sheet, I'll update here.

**Comment by ahmedkrmn** [comment]
Hi @marionschleifer, are there any specific examples that we need to add?

**Comment by marionschleifer** [comment]
@ahmedkrmn just look at the examples in the page and add examples along these lines 🙂

**Comment by vbansal20** [comment]
pls assign me too !!

**Comment by christopheroye** [comment]
Please assign me to this project. I would love to work on this!
